templates/editor/boot-partial.html

```diff
@@ -14,5 +14,15 @@
         }
       };
     {% endif %}
+
+    if (!window.onPiskelReady) {
+      window.onPiskelReady = [];
+    }
+
+    window.onPiskelReady.push(function() {
```

Comment:
If Piskel was already loaded when we reach this point, the callback will never be called.
To be safe it would be nice to simply check if ($ && $.subscribe) is already available, and in this case, directly execute the callback.